bootstrap/exherbo.sh

```diff
@@ -0,0 +1 @@
+_exherbo_common.sh
```

Comment:
Sorry, #1221 gave a bad example. Please use option 3. As a bonus you could fix Gentoo script! :)